src/ziggurat/init.clj

```diff
@@ -44,8 +44,13 @@
             #'cpool/channel-pool} args))
 
 (defn- start-rabbitmq-consumers [args]
-  (start-rabbitmq-connection args)
-  (messaging-consumer/start-subscribers (get args :stream-routes) (get args :batch-routes)))
+  (start* #{#'consumer-connection
+            #'producer-connection
+            #'cpool/channel-pool
+            #'messaging-consumer/consumers} args))
+
+(defn- stop-rabbitmq-consumers []
```

Comment:
<img width="830" alt="Screenshot 2022-09-13 at 5 28 54 PM" src="https://user-images.githubusercontent.com/13599208/189895324-9df15954-02a7-40d5-9ffa-7ed7cdc1078e.png">
@mjayprateek When left on mount, we see that the consumer connection is stopped before the subscribers and hence we are left with the attached error.
While starting, since we have to make the queues, we first explicitly start rabbitmq connection. And hence subscribers naturally start after the connection.

Reply:
Keeping consumer-connection state in consumers state makes it work. Mount works as expected.